locales/en/product-registration.json

```
@@ -0,0 +1,32 @@
+{
+  "register": "REGISTER",
+  "productImage": "Product Image(s)",
+  "addThreeImages": "Add up to 3 images",
+  "name": "Name",
+  "price": "Price (THB)",
+  "color": "Color",
+  "size": "Size",
+  "category": "Category",
+  "deliveryOption": "Delivery Option",
+  "footwear": "Footwear",
+  "shirt": "Shirts",
+  "accessory": "Accessories",
+  "postal": "Postal",
+  "meetUp": "Meet up",
+  "both": "Postal & Meet up",
+  "description": "Description",
+  "submit": "SUBMIT",
+  "namePlaceholder": "E.g. Nike Air Max, Adidas Ultraboost",
+  "pricePlaceholder": "E.g. 290, 3500.50",
```

This one is not suppose to have a decimal amount of money from our previous discussion to input only an integer number.
```suggestion
  "pricePlaceholder": "E.g. 290",
```